### PR TITLE
edit configuration for deploy 

### PR DESCRIPTION
**Pull request by Saeed99Madi**
related to #97

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,11 +57,11 @@
...
     "nodemon": "^2.0.20",
     "prettier": "2.7.1",
-    "supertest": "^6.3.0",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.8.4"
+    "supertest": "^6.3.0"
   },
   "dependencies": {
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4",
```
Comment (Amoodaa) [inline package.json:64]: wrong

File: package.json [hunk 2]
```diff
@@ -82,5 +84,8 @@
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.24.0",
     "ts-jest": "^29.0.3"
-  }
+  },
+  "optionalDependencies": {
+    "fsevents": "^2.0.7"
+}
```
Comment (Amoodaa) [inline package.json:90]: unneeded without mac

File: package.json [hunk 3]
```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "test": "cross-env NODE_ENV=test jest --runInBand",
     "buildDB": " cross-env NODE_ENV=development ts-node src/db/build.ts",
+    "build-db": " cross-env NODE_ENV=production ts-node src/db/build.ts",
```
Comment (Amoodaa) [inline package.json:9]: this should be
```suggestion
    "build-db": " cross-env NODE_ENV=production node dist/db/build.js",
```

**Review by dev-mohmmed-alshorafa — APPROVED** [review]
good job man

File: package.json (hunk 2 above)
Comment (Saeed99Madi) [reply package.json]: great

File: package.json (hunk 3 above)
Comment (Saeed99Madi) [reply package.json:9]: The deploy does not work without it

**Review by AseelL — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (Amoodaa) [reply package.json:9]: make sure your steps are in order so
build and then db build the node version

**Review by Amoodaa — APPROVED** [review]

**Pull request merged**